org_fedora_oscap/rule_handling.py

```diff
@@ -668,3 +683,78 @@ def eval_rules(self, ksdata, storage, report_only=False):
             return []
 
     # nothing to be reverted for now
+
+
+class KdumpRules(RuleHandler):
+    """Simple class holding data from the rules affecting the kdump addon."""
+
+    def __init__(self):
+        """Constructor setting the initial value of attributes."""
+
+        self._kdump_enabled = None
+        self._kdump_default_enabled = None
+
+    def kdump_enabled(self, kdenabled):
+        """Enable or Disable Kdump"""
+
+        if kdenabled == True:
+            self._kdump_enabled = True
+        if kdenabled == False:
+            self._kdump_enabled = False
+
+    def __str__(self):
+        """Standard method useful for debugging and testing."""
+
+        ret = "kdump"
+
+        if self._kdump_enabled == True:
+            ret += " --enable"
+
+        if self._kdump_enabled == False:
+            ret += " --disable"
```

Comment:
None is not handled here, not sure if it's okay or not

Reply:
@mpreisler that is fine. We don't care about `None` here.

Reply:
Not sure why not to use
```
if self._kdump_enabled:
  ret += " --enable"
else:
  ret += " --disable"

Reply:
@dahaic because `--disable` or `--enable` may not be selected at all. This logic is handled the same way for other classes as well in `rule_handling.py`

Reply:
@dahaic it's essentially a tribool